### PR TITLE
bookmark page update

### PR DESCRIPTION
**Pull request by hemantkasat**

### CODE REVIEW

**Comment by hemantkasat** [comment]
what is this error   and how i should resolve it ??

File: gradle.properties [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+#
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+#Mon Jan 04 10:32:12 IST 2016
+systemProp.https.proxyPort=8080
+systemProp.http.proxyHost=proxy.iiit.ac.in
```
Comment (mananwason) [inline gradle.properties]: Remove these @hemantkasat

**Comment by mananwason** [comment]
@hemantkasat You need to restart the travis build

**Comment by mananwason** [comment]
@hemantkasat are you working on this?

**Comment by mananwason** [comment]
@hemantkasat I am closing this for now.

**Pull request closed without merging**